### ISSUE TITLE
[Accessibility]: LineChart / BarChartStacked [role] values are not valid

### ISSUE THREAD

**Issue by mschmdb** [carbon-design-system/carbon-charts#1526]
### Name

Matze

### Are you an IBM employee?

- [ ] Yes

### Contact Details

m@neun.dev

### Environment

Browser

### What happened? What did you expect to happen instead?

Tested my website with integrated carbon Linechart and BarchartStacked with Lighthouse and I got: **[role] values are not valid** for both types of charts.
 Following failing elements:
**LineChart:**
`left ticks
<g class="ticks" role="graphics-object group" aria-label="left ticks" transform="translate(66.4296875, 0)" fill="none" font-size="10" font-family="sans-serif" text-anchor="end">`

`bottom ticks
<g class="ticks" role="graphics-object group" aria-label="bottom ticks" transform="translate(0, 277.9799995422363)" fill="none" font-size="10" font-family="sans-serif" text-anchor="middle">`
**BarChartStacked:**
`left ticks
<g class="ticks" role="graphics-object group" aria-label="left ticks" transform="translate(61.0859375, 0)" fill="none" font-size="10" font-family="sans-serif" text-anchor="end">`

`bottom ticks
<g class="ticks" role="graphics-object group" aria-label="bottom ticks" transform="translate(0, 251.97999954223633)" fill="none" font-size="10" font-family="sans-serif" text-anchor="middle">`

### What WCAG 2.1 checkpoint does the issue violate?

[role] values are not valid

### Version

"@carbon/charts": "^1.6.14"

### What priority level would this issue be in your opinion?

P0

**Comment by theiliad** [comment]
Hi @mschmdb, what part of these roles aren't valid? Does Lighthouse offer you any more details?

https://www.w3.org/TR/graphics-aria-1.0/#graphics-object

**Comment by mschmdb** [comment]
Hey @theiliad after studying the link you provided i guess Lighthouse is wrong. Thanks for that.

**Comment by theiliad** [comment]
> Hey @theiliad after studying the link you provided i guess Lighthouse is wrong. Thanks for that.

I've seen other tools identify these roles as violations. In 1 case I remember it was because of a lack of full support for SVGs
